frontend/awx/resources/inventories/InventoryForm.tsx

```diff
@@ -310,13 +310,13 @@ export function useInventoryFormDetailLabels() {
 
   return {
     labels: t(
-      `Optional labels that describe this inventory, such as 'dev' or 'test'. Labels can be used to group and filter inventories and completed jobs.`
+      `Optional labels that describe this inventory, such as 'dev' or 'test.' Labels can be used to group and filter inventories and completed jobs.`
```

Comment:
In this case I feel like it should be after the quote because we're referring to a specific example of a name. The quotes in this case are just meant to wrap around the names of the inventories. Can we change it back?

Reply:
Yes, I can change it back